### PR TITLE
Jira v2 - Incidents mirror-in

### PR DESCRIPTION
**Pull request by GalRabin**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/16193

## Description
Adding support for mirror incident into demisto server (If issue change in Jira it will be reflected in demisto incident).
Implementation contains:
1. Mapper-in classifier.
2. New command `get-remote-data`.

## Screenshots
Paste here any images that will help the reviewer
image

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Documentation

### CODE REVIEW

**Comment by GalRabin** [comment]
@yaakovi  Reminder

File: Packs/Jira/Integrations/JiraV2/JiraV2.py [hunk 1]
```diff
@@ -645,6 +669,41 @@ def fetch_incidents(query, id_offset, fetch_by_created=None, **_):
...
+    # Get raw response on issue ID
+    _, _, issue_raw_response = get_issue(issue_id=id)
+
+    # Timestamp - Issue last modified in jira server side
+    jira_modified_date: datetime = parse_date_string(dict_safe_get(issue_raw_response,
+                                                                   ['fields', 'updated'], "", str))
+    # Timestamp - Issue last sync in demisto server side
+    incident_modified_date: datetime = parse_date_string(lastUpdate)
+
+    # Update incidents only if issue modified in jira server side after last sync
```
Comment (yaakovi) [inline Packs/Jira/Integrations/JiraV2/JiraV2.py:696]:
```suggestion
    # Update incident only if issue modified in Jira server-side after the last sync
```

**Review by yaakovi — COMMENTED** [review]
please schedule a short demo on Sunday

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml [hunk 2]
```diff
@@ -299,13 +309,35 @@ script:
       description: The ID or key of the issue.
     description: Deletes an issue in Jira.
   - name: jira-get-id-offset
+    description: Get id offset from begining.
```
[the remark this answers is not in the document]
Comment (GalRabin) [reply Packs/Jira/Integrations/JiraV2/JiraV2.yml:312]: I just completed missing details there, The validation fails the build.
The command used to fetch the first issue exists.

**Review by yaakovi — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@yaakovi Doc review is done. YML still needs to be updated with the fix from https://github.com/demisto/etc/issues/29648

**Pull request merged**